apps/linkedin-to-notion/src/contents/utils/utils.ts

```diff
@@ -0,0 +1,24 @@
+const customQuerySelector = (selector: string, searchedText: string): boolean => {
```

Comment:
Not a big fan of the `utils/utils.ts`
I'd rather go for `shared/waitForElementToExist.ts`